app/views/hiring_staff/vacancies/review_vacancy_sections/_review_supporting_documents.html.haml

```diff
@@ -7,10 +7,15 @@
 -#     %span.govuk-visually-hidden the supporting documents
 
 %dl.app-check-your-answers.app-check-your-answers--short
-  - @vacancy.documents.each_with_index do |document, index|
-    - index_from_one = index + 1
+  - if @vacancy.documents.none?
     .app-check-your-answers__contents
-      %dt.app-check-your-answers__question#supporting_documents
-        = "Document #{index_from_one}"
-      %dd.app-check-your-answers__answer
-        = document[:name]
+      %dt#supporting_documents{class: "govuk-!-font-weight-bold"}
+        = "No documents uploaded"
```

Comment:
Probably this text should come from en.yml, both here and in the test, to enable easy changes.